Gravity-Notes/src/Components/Notes-card.js/Notecard.js

```diff
@@ -5,14 +5,43 @@ import {BiImage} from "react-icons/bi"
 import {BiArchiveIn} from "react-icons/bi"
 import {MdMoreVert} from "react-icons/md"
 import "./Notecard.css";
-export const Notecard = ({title, content, key})=>{
+import { useAuth } from "../../Contexts/Auth-context"
+import { useNotes } from "../../Contexts/NotesAction-context"
+import axios from "axios"
+
+
+
+
```

Comment:
Remove unwanted spaces for better readability - Just Personal Suggestion (you can overlook it)